src/rsdiv/diversity/mmr.py

```diff
@@ -0,0 +1,45 @@
+from typing import Optional, Sequence
+
+import numpy as np
+import numpy.ma as ma
+
+from .base import BaseReranker
+
+
+class MaximalMarginalRelevance(BaseReranker):
```

Comment:
I've implemented a `pytest` template under `rsdiv/test/recommenders/`. Try to rebase the `develop` and pass the `pytest` checks.